src/bandersnatch/verify.py

```diff
@@ -117,6 +117,7 @@ async def verify(
     args: argparse.Namespace,
     executor: concurrent.futures.ThreadPoolExecutor | None = None,
     releases_key: str = "releases",
+    sha256_check: bool = True
```

Comment:
This should be more generic as I'm sure we'll move from sha256 one day ... But this is outside this PR

Reply:
yes, but I didnt find a better name, will try later 😄